src/database.py

```diff
@@ -0,0 +1,178 @@
+import sqlite3
+from dataclasses import dataclass, field
+from typing import Optional, Type, Union, Tuple, List
+from types import TracebackType
+
+class DbManager:
```

Comment:
Use an ORM instead of coding your request. It's safer in order to prevent Sql Injection. 
Alchemy is a good start

Reply:
now implemented after https://github.com/atyrode/vite/pull/12/commits/9821d35af01dc0f63374c413c59476fe5c12cf9c, thanks for mentionning SQLAlchemy, I had no idea this way a thing and I won't be looking back, feels like a must now 😄